requirements-minimal.txt

```diff
@@ -10,9 +10,7 @@ certifi
 choice >= 0.1
 cookiecutter >= 1.4.0
 croniter
-# Don't update this unless you have confirmed the client works with
-# the Docker version deployed on PaaSTA servers
-docker-py >= 1.2.3
+docker
```

Comment:
none of 
* https://github.com/docker/docker-py/releases/tag/2.0.0
* https://github.com/docker/docker-py/releases/tag/3.0.0
* https://github.com/docker/docker-py/releases/tag/4.0.0
* https://github.com/docker/docker-py/releases/tag/5.0.0

appear to mention being incompatible with our current docker version

Reply:
that said, i can also run this on some infrastage boxes before merging this to make sure that we're not being misled by the tests passing on devboxes/GHA

Reply:
fwiw: `docker` the library not working wouldn't cause an outage - it would just break oom logging and some cleanup cronjobs. 

this was a lot scarier in the mesos days :)

Reply:
as far as i can tell with my manual testing on an infrastage box/my devbox, this works as expected :)